lib/NetPacksLib.cpp

```diff
@@ -1304,6 +1304,9 @@ DLL_LINKAGE void StartAction::applyGs( CGameState *gs )
 {
 	CStack *st = gs->curB->getStack(ba.stackNumber);
 
+	if(vstd::contains(st->state, EBattleStackState::DEFENDING_ANIM))

```

Comment:
This one cause crashes:

* When quick battle is enabled (no animations going to happen at all).
* When spell casted on first turn in battle.